src/ctia/task/migration/migrations.clj

```diff
@@ -162,4 +242,6 @@
                  pluralize-target)
    :1.0.0 (comp (append-version "1.0.0")
                 (rename-observable-type "pki-serial" "pki_serial")
-                simplify-incident)})
+                simplify-incident)
+   :1.1.0 (comp (append-version "1.1.0")
+                migrate-action-data)})
```

Comment:
Could you test this migration, please?
You have only one chance with that. In case of failure, the best case is that it fails and does not migrate the documents and we do not restart on the new documents. The worst case is that we switch on the newly created index with bad documents, insert new investigation entities, discover the bug and have to write a new migration.

Reply:
I added tests for the migration and the primary unit of work `derive-action-data`.